usaon_benefit_tool/routes/login.py

```diff
@@ -1,30 +1,34 @@
+import os
+
 from flask import Blueprint, redirect, url_for
-from flask_dance.contrib.google import google
+from flask_dance.consumer import oauth_authorized
+from flask_dance.contrib.google import google, make_google_blueprint
 from flask_login import login_user
 
 from usaon_benefit_tool.util.db.user import ensure_user_exists
 
 login_bp = Blueprint('login', __name__, url_prefix='/login')
+google_bp = make_google_blueprint(
+    client_id=os.getenv('USAON_BENEFIT_TOOL_GOOGLE_CLIENT_ID'),
+    client_secret=os.getenv('USAON_BENEFIT_TOOL_GOOGLE_CLIENT_SECRET'),
+    scope=["profile", "email"],
+)
 
 
 @login_bp.route("")
 def login():
     if not google.authorized:
         return redirect(url_for("google.login"))
-    resp = google.get("/oauth2/v2/userinfo")
-    assert resp.ok, resp.text
-
-    user = ensure_user_exists(resp.json())
-    login_user(user)
```

Comment:
@rmarow I think our problem is here! When the user hits this endpoint, they get redirected to the google login endpoint and never reach this code that actually logs them in from the application's perspective (`login_user`)